src/components/new-request/search.vue

```diff
@@ -103,11 +103,44 @@
       <!-- Change Name flow -->
       <template v-else-if="isChangeName">
         <BusinessLookupFetch />
-
+        <v-dialog  v-if="isSocietyDisabled" v-model="showSocietiesInfo"
```

Comment:
I think you don't need the v-if here, because you are already controlling the visibility of this dialog with the v-model property.

Reply:
Fixed, thanks.